packages/smart/contracts/turbo/AMMFactory.sol

```diff
@@ -95,12 +118,17 @@ contract AMMFactory is BNum {
 
         pools[address(_marketFactory)][_marketId] = _pool;
 
+        uint256 nextPID = masterChef.poolLength();
```

Comment:
Our style is to precede local vars with an underscore unless they're forloop vars.